src/NServiceBus.Transport.Msmq/MsmqUtilities.cs

```diff
@@ -207,6 +207,8 @@ static void AssignMsmqNativeCorrelationId(OutgoingMessage message, Message resul
         const string DIRECTPREFIX_TCP = "DIRECT=TCP:";
         internal const string PRIVATE = "\\private$\\";
 
+        static readonly XmlReaderSettings HeaderSerializerXmlReaderSettings = new XmlReaderSettings { CheckCharacters = false };
```

Comment:
Are we sure these are the correct settings? Previous version have been using this overload right?

https://github.com/dotnet/runtime/blob/5535e31a712343a63f5d7d796cd874e563e5ac14/src/libraries/System.Private.Xml/src/System/Xml/Serialization/XmlSerializer.cs#L423

which uses

`new XmlReaderSettings() { IgnoreWhitespace = true }`

Reply:
Ah I double checked and we used the XmlReaderSettings directly https://github.com/Particular/NServiceBus.Transport.Msmq/blob/release-1.2/src/NServiceBus.Transport.Msmq/MsmqUtilities.cs#L104-L107